tools/jobs.php

```diff
@@ -0,0 +1,420 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @file tools/jobs.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class jobs
+ * @ingroup tools
+ *
+ * @brief CLI tool to list, iterate and purge queued jobs on database
+ */
+
+namespace PKP\tools;
+
+use APP\facades\Repo;
+use Exception;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Console\Concerns\InteractsWithIO;
+use Illuminate\Console\OutputStyle;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use PKP\cliTool\CommandLineTool;
+use Symfony\Component\Console\Exception\CommandNotFoundException;
+use Symfony\Component\Console\Exception\InvalidArgumentException as CommandInvalidArgumentException;
+use Symfony\Component\Console\Exception\LogicException;
+use Symfony\Component\Console\Helper\Helper;
+use Symfony\Component\Console\Helper\TableCell;
+use Symfony\Component\Console\Helper\TableCellStyle;
+use Symfony\Component\Console\Input\StringInput;
+
+use Symfony\Component\Console\Output\StreamOutput;
+use Throwable;
+
+define('APP_ROOT', dirname(dirname(dirname(dirname(__FILE__)))));
+require(APP_ROOT . '/tools/bootstrap.inc.php');
+
+class commandInterface
+{
+    use InteractsWithIO;
+
+    public function __construct()
+    {
+        $output = new OutputStyle(
+            new StringInput(''),
+            new StreamOutput(fopen('php://stdout', 'w'))
+        );
+
+        $this->setOutput($output);
+    }
+
+    public function errorBlock(array $messages = [], ?string $title = null): void
+    {
+        $this->getOutput()->block(
+            $messages,
+            $title,
+            'fg=white;bg=red',
+            ' ',
+            true
+        );
+    }
+}
+
+class commandJobs extends CommandLineTool
```

Comment:
This class needs to be localized; I'd suggest putting the relevant locale keys in `lib/pkp/locale/en_US/admin.po` where you can't find already-existing ones.

Reply:
I don't think it's possible to i18n the tools. I've tried and the result was unsuccessful.

Reply:
![Screenshot from 2022-01-19 11-50-42](https://user-images.githubusercontent.com/1243634/150155118-afb68383-ac9d-484e-8cca-175a0af8b14a.png)

Reply:
Thank you @NateWr!